unittest/gunit/keyring/CMakeLists.txt

```diff
@@ -56,7 +56,6 @@ LIST(APPEND SRC_FILES
   ${CMAKE_SOURCE_DIR}/plugin/keyring/checker/checker_ver_2_0.cc
   ${CMAKE_SOURCE_DIR}/unittest/gunit/keyring/buffered_file_io_10.cc
   ${CMAKE_SOURCE_DIR}/unittest/gunit/keyring/psi_memory_key.cc
-  ${CMAKE_SOURCE_DIR}/unittest/gunit/keyring/uuid.cc
```

Comment:
Cannot find if gunit/keyring/uuid.cc was removed as well?

Reply:
Yes, the keyname uuid support was related to keyring encryption.

Reply:
I don't see this file being deleted in the change. Should we delete it?

Reply:
Yes, it is deleted in the linear branch version, but I missed the removal in this single commit PR. I'll delete it.